java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -106,7 +106,7 @@
       javaType: "ArrowBuf",
       boxedType: "ArrowBuf",
       minor: [
-        { class: "IntervalDay", millisecondsOffset: 4, friendlyType: "Period", fields: [ {name: "days", type:"int"}, {name: "milliseconds", type:"int"}] }
+        { class: "IntervalDay", millisecondsOffset: 4, friendlyType: "Duration", fields: [ {name: "days", type:"int"}, {name: "milliseconds", type:"int"}] }
```

Comment:
`java.time.Duration` seems to be the corresponding class to use now. It can represent interval in days and time. Period can only do year, month, days.